### ISSUE TITLE
WIFI

### ISSUE THREAD

**Issue by dkim777** [VolosR/TTGOWeatherStation#11]
Is there any way to enter WIFI setting from smartphone or on screen device itself without entering them in the code?

**Comment by soharddbwarez** [comment]
Yes there is a way and I'm working on getting that added to the project, I've made a lot of changes to the code already so if you want to follow the work I'm doing you can find the fork of this in my repository here: https://github.com/soharddbwarez/MyArduinoProjects/tree/master/WeatherStation

Best regards

**Comment by JuniorPolegato** [comment]
> Is there any way to enter WIFI setting from smartphone or on screen device itself without entering them in the code?

Hi!

I did this project to provide a user interface to configure your ESP project via web browser [ESPUserConnection](https://github.com/JuniorPolegato/ESPUserConnection).

In this fork [TTGOWeatherStation](https://github.com/JuniorPolegato/TTGOWeatherStation) I incorporated a user interface via web browser, so you don't need anymore change the code to select new WiFi, key or city.
